roles/repository/templates/xorigins_macros.j2

```diff
@@ -1,4 +1,4 @@
-{% macro compute(type, urls) -%}
+{% macro compute(host, type, urls) -%}
```

Comment:
I unfortunately had to change the macro interface as the regular {{ inventory_hostname }} variable seem to not be available whithin jinja imports :'(